scss/grid/_position.scss

```diff
@@ -8,7 +8,7 @@
 
 /// Reposition a column.
 ///
-/// @param {Number} $position - Direction and amount to move. The column will move equal to the width of the column count specified. A positive number will push the column to the right, while a negative number will pull it to the left.
+/// @param {Number} $position - Direction and amount to move. The column will move equal to the width of the column count specified. A positive number will push the column to the right, while a negative number will pull it to the left. Set to "center" (without quotes) to center the column.
```

Comment:
:+1: Thanks.

I have some proposals:
- remove `(without quotes)`, it is not necessary
- change the argument type to `{Number|Keyword}`